velox/serializers/tests/PrestoSerializerTest.cpp

```diff
@@ -1339,6 +1373,25 @@ TEST_P(PrestoSerializerTest, encodedRoundtrip) {
   }
 }
 
+TEST_P(PrestoSerializerTest, opaqueBatchVectorSerializer) {
```

Comment:
can you also add a test to ensure that opaque type will throw an error if used with Iterative serializer and if estimateSerializedSize is used